src/core/Global.ttslua

```diff
@@ -945,6 +951,7 @@ function onClick_defaultSettings()
   -- clean reset of variable
   optionPanel = {
     useSnapTags = true,
+    PlayAreaSnapTags = false,
```

Comment:
This should default true, right?

Reply:
Was hesitant to do that because custom campaigns are way more prominent than custom player cards - so it would happen quite often that people would need to disable this

Reply:
You're checking in the snap points with the location set though, which means the default is out of sync with what the mod loads with.

Generally we've been optimizing the default state of the mod for official content, with options to change that for custom.  I'm not completely against this being different, but I do think we should be consistent where possible.

Reply:
Alright, changed it